app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= javascript_include_tag 'recipe' %>
+<div>
+  <div class="container-fluid">
+    <div class="row justify-content-center">
+      <div class="col-md">
+        
+        <div class="d-flex justify-content-between mt-4">
+          <h2 class="food_title mb-4">Recipe</h2>
+         <%= form_for @recipe, id: "recipe_form" do |f| %>
```

Comment:
- [ ] I just noticed that the recipe_id is only displayed in our terminal but not on our screen, as shown in the wireframe; please make the recipe id visible on the UI so that the user can easily recognize the recipe, as demonstrated in the wireframe. :wink: 
![Screenshot from 2023-04-01 20-21-29](https://user-images.githubusercontent.com/88894525/229307996-3ef97a69-e4a5-4744-9036-48d8383cb3ad.png)

Your design 
![Screenshot from 2023-04-01 20-25-04](https://user-images.githubusercontent.com/88894525/229308103-22d1ca7e-6fc2-4324-abdc-55246355d5a3.png)



In console



![Screenshot from 2023-04-01 20-15-33](https://user-images.githubusercontent.com/88894525/229307890-78420c72-8d91-456e-b8c4-634fd6f334c5.png)